napalm_ios/ios.py

```diff
@@ -354,6 +354,8 @@ def commit_config(self):
                 raise ReplaceConfigException("Candidate config file does not exist")
             if self.auto_rollback_on_error:
                 cmd = 'configure replace {} force revert trigger error'.format(cfg_file)
+            elif confirmed:
+                cmd = 'configure replace bootflash:{} force time {}'.format(cfg_file, confirmed)
```

Comment:
I have hardcoded as I have noticed that it wasn't able to detect the file system and for this command it seems that is mandatory -- but I will double check this & fix.